examples/src/main/scala/com/stratio/crossdata/examples/driver/StreamingSqlExample.scala

```diff
@@ -53,17 +48,17 @@ object StreamingSqlExample extends App with CassandraDefaultConstants with Strea
 
   try{
     // Imports tables from Cassandra cluster
-    driver.syncQuery(SQLCommand(importQuery))
+    driver.sql(importQuery).waitForResult()
 
     // Creates the ephemeral table (streaming input) receiving data from Kafka
-    driver.syncQuery(SQLCommand(createEphemeralTable))
+    driver.sql(createEphemeralTable).waitForResult()
 
     // Adds a streaming query. It will be executed when the streaming process is running
-    driver.syncQuery(SQLCommand(s"SELECT count(*) FROM $EphemeralTableName WITH WINDOW 5 SECS AS outputTopic"))
+    driver.sql(s"SELECT count(*) FROM $EphemeralTableName WITH WINDOW 5 SECS AS outputTopic")
    
 
     // Starts the streaming process associated to the ephemeral table
-    driver.syncQuery(SQLCommand(s"START $EphemeralTableName"))
+    driver.sql(s"START $EphemeralTableName")
```

Comment:
Does it make sense to add "waitForResult" here? Does it make sense to start the ephemeral table before the first select?